Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use an official Python runtime as a parent image
+FROM python:3.7
+
+RUN pip install pipenv
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+ADD . /app
+
+# Install env
+
+RUN pipenv install
```

Comment:
Copy Pipfile.lock first, install the dependencies from it, then copy everything else. This will eliminate unnecessary container rebuilds, dependencies will be install strictly from Pipfile.lock, not from Pipfile. Do not use the ADD command, use the COPY command. There you go:
```dockerfile
COPY Pipfile.lock ./

RUN pipenv install --ignore-pipfile --keep-outdated

COPY . ./
```